src/kernel/bitmap.zig

```diff
@@ -666,10 +545,10 @@ test "setFirstFree multiple bitmaps" {
     // Allocate the entirety of the first bitmap
     var entry: u32 = 2;
     var expected: u8 = 7;
-    while (entry < Bitmap(u8).ENTRIES_PER_BITMAP) {
+    while (entry < @TypeOf(bmp).ENTRIES_PER_BITMAP) {
```

Comment:
Am being picky, but would it be nicer to have:
```zig
const BmpTy = Bitmap(false, u8, null);
var bmp = try BmpTy.init(9, std.testing.allocator);
```
so you don't have to call `@TypeOf` all the time. But up to you.

Reply:
Yeah that looks good to me.